torch/onnx/symbolic_opset9.py

```diff
@@ -4482,9 +4482,9 @@ def try_mask_to_index(index):
             rank = symbolic_helper._get_tensor_rank(self)
             if rank is None:
                 raise NotImplementedError(
-                    "Unsupported aten::index operator of advanced indexing on tensor of unknown rank, "
-                    + "try turning on shape and type propagate during export: "
-                    + "torch.onnx._export(..., propagate=True)."
+                    "Unsupported aten::index operator of advanced indexing on tensor of unknown rank. "
+                    "Try turning on shape inference during export: "
+                    "torch.onnx._export(..., onnx_shape_inference=True)."
```

Comment:
While you are at it, I think `onnx_shape_inference` is turned on by default. So this recommendation is no longer valid to users who use the `export` API. This option can only be turned _off_ with the internal `_export` api.

Reply:
Although it is on by default, it might be off and this message alerts the user to the fact that shape inference can fix this in an actionable way.